pkg/manager/impl/execution_manager_test.go

```diff
@@ -198,6 +198,13 @@ func getMockRepositoryForExecTest() repositories.RepositoryInterface {
 
 func TestCreateExecution(t *testing.T) {
 	repository := getMockRepositoryForExecTest()
+	repository.ProjectRepo().Create(context.Background(), models.Project{
```

Comment:
I think you need to mock out the Get call to return a project with the proper labels

Reply:
fixed, tested and working:
Running tool: /usr/local/opt/go/libexec/bin/go test -timeout 30s github.com/lyft/flyteadmin/pkg/manager/impl -run ^TestCreateExecution$

ok  	github.com/lyft/flyteadmin/pkg/manager/impl	0.503s